examples/customVideoRenderer/src/main/java/com/twilio/video/examples/customrenderer/CustomRendererVideoActivity.java

```diff
@@ -0,0 +1,108 @@
+package com.twilio.video.examples.customrenderer;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.twilio.video.CameraCapturer;
+import com.twilio.video.LocalMedia;
+import com.twilio.video.LocalVideoTrack;
+import com.twilio.video.VideoView;
+
+/**
+ * This example demonstrates how to implement a custom custom renderer. Here we render the contents
+ * of our {@link CameraCapturer} to a video view and to a snapshot renderer which allows user to
```

Comment:
to a video view. Additionally we implement a custom renderer, dubbed the 'snapshot renderer' which allows...